operational/azure/azure_migrate/risc-azure-migrate.pt

```diff
@@ -0,0 +1,309 @@
+name "Azure Migrate Integration"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy will collect the resources from a RISC Foundations assessment and seed Azure Migrate with the discovered servers. See the [README](https://github.com/rightscale/policy_templates/tree/master/operational/azure/azure_migrate) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
include details about action

Reply:
that's what "seed Azure Migrate with the discovered servers" is